toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexServiceImpl.kt

```diff
@@ -125,24 +125,32 @@ class CratesLocalIndexServiceImpl
         this.state = state
     }
 
+    @Throws(CratesLocalIndexException::class)
```

Comment:
Let's annotate the interface method instead.

Also, I'd recommend adding a descriptive doc comment like
```Kotlin
/**
 * @throws CratesLocalIndexException if ...
 * @return CargoRegistryCrate ...
 * @return `null` ...
 */
```